package toPlugin;

import "pluginInformation.proto";


message System {
	enum EventType {
		kRequestStop = 0;
	}
	required EventType type = 1;
}

message BindingQuery {
	required string query = 1;
}

message DeviceCommand {

}

message ExtraCommand {

}



message msg {
	optional string error = 1;

	oneof OneOf {
		System system = 2;
		pbPluginInformation.Information pluginInformation = 3;
		BindingQuery bindingQuery = 4;
		DeviceCommand deviceCommand = 5;
		ExtraCommand extraCommand = 6;
	}
}

